facebookmessenger-definition/src/main/java/ai/labs/facebookmessenger/endpoint/IFacebookEndpoint.java

```diff
@@ -0,0 +1,24 @@
+package ai.labs.facebookmessenger.endpoint;
+
+import io.swagger.annotations.Api;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Path("/channels/facebook")
+@Api(description = "channels")
+public interface IFacebookEndpoint {
```

Comment:
You seem to have 2 FacebookEndpoints:
* `ai.labs.facebookmessenger.endpoint.FacebookEndpoint`
* `ai.labs.core.rest.internal.FacebookEndpoint`

I guess this is the "right one"?